src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/index.tsx

```diff
@@ -232,7 +232,7 @@ const columns: ColumnDef<Sample, any>[] = [
       </SortableHeader>
     ),
     cell: ({ getValue }) => {
-      const { lineage } = getValue();
+      const lineage = getValue()?.lineage;
```

Comment:
Chrome wasn't letting me render the new table because getValue was undefined and js ate it when trying to destructure.  I'm not entirely sure what changed (I assume this must have worked before).